tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -403,13 +403,12 @@ public function testFetchColumn()
         $this->assertSame($result, $conn->fetchColumn($statement, $params, $column, $types));
     }
 
-    public function testConnectionIsClosed()
```

Comment:
I don't believe this is a valid test, which is what I'm arguing here: https://github.com/doctrine/dbal/pull/784/files#r23499338. I would like to remove this test - I was using that PR simply to abuse Travis to show how the test was passing for the wrong reason.